src/core_plugins/kibana/public/management/sections/objects/change_index_modal.js

```diff
@@ -0,0 +1,207 @@
+        <KuiTableRow key={key}>
+          <KuiTableRowCell>
+            {indexPatternId}
+          </KuiTableRowCell>
+          <KuiTableRowCell>
+            {objects.length}
+          </KuiTableRowCell>
+          <KuiTableRowCell>
+            {sample}
+          </KuiTableRowCell>
+          <KuiTableRowCell>
+            <select
+              className="kuiSelect kuiSelect--medium"
+              data-test-subj="managementChangeIndexSelection"
+              value={this.state.objects[indexPatternId].newIndexPatternId}
+              onChange={this.onIndexChange.bind(this, indexPatternId)}
+            >
+              <option value={indexPatternId}>-- None --</option>
+              {this.props.indices.map((index, i) => {
+                return (
+                  <option key={i} value={index.id}>
+                    {index.get('title')}
+                  </option>
+                );
+              })}
+            </select>
+          </KuiTableRowCell>
+        </KuiTableRow>
+      );
+    });
+
+    return (
+      <KuiModalOverlay>
+        <KuiModal
+          data-tests-subj="managementChangeIndexModal"
+          aria-label="Index does not exist"
+          className="managementChangeIndexModal"
+          onKeyDown={this.onKeyDown}
+        >
+          <KuiModalHeader>
+            <KuiModalHeaderTitle>
+              Index Pattern Conflicts
+            </KuiModalHeaderTitle>
+          </KuiModalHeader>
+          <KuiModalBody>
+            <KuiModalBodyText>
+              <p>The following saved objects use index patterns that do not exist.</p>
+            </KuiModalBodyText>
+            <KuiControlledTable>
+              <KuiToolBar>
+                <KuiToolBarSection>
+                  { totalIndexPatterns > perPage
+                    ?
+                      <KuiPager
+                        startNumber={page + 1}
+                        hasPreviousPage={page >= 1}
+                        hasNextPage={page < totalIndexPatterns}
+                        endNumber={Math.min(totalIndexPatterns, page + perPage)}
+                        totalItems={totalIndexPatterns}
+                        onNextPage={() => this.setState({ page: page + 1 })}
+                        onPreviousPage={() => this.setState({ page: page - 1 })}
+                      />
+
+                    : null
+                  }
+                </KuiToolBarSection>
+              </KuiToolBar>
+              <KuiTable className="kuiVerticalRhythm">
+                <KuiTableHeader>
+                  <KuiTableHeaderCell width="300">
+                    ID
+                  </KuiTableHeaderCell>
+                  <KuiTableHeaderCell width="50">
+                    Count
+                  </KuiTableHeaderCell>
+                  <KuiTableHeaderCell>
+                    Sample of Affected Objects
+                  </KuiTableHeaderCell>
+                  <KuiTableHeaderCell width="200">
+                    New Index
```

Comment:
Do we want this to say "New index pattern" instead? (Note the sentence casing too).